core/experiments/manager.php

```diff
@@ -405,18 +405,6 @@ private function add_default_features() {
 			],
 		] );
 
-		$this->add_feature( [
-			'name' => 'e_swiper_latest',
-			'title' => esc_html__( 'Upgrade Swiper Library', 'elementor' ),
-			'description' => esc_html__( 'Prepare your website for future improvements to carousel features by upgrading the Swiper library integrated into your site from v5.36 to v8.45. This experiment includes markup changes so it might require updating custom code and cause compatibility issues with third party plugins.', 'elementor' ),
-			'release_status' => self::RELEASE_STATUS_STABLE,
-			'new_site' => [
-				'default_active' => true,
-				'minimum_installation_version' => '3.11.0',
-			],
-			'default' => self::STATE_INACTIVE,
-		] );
-
```

Comment:
@TzviRabinovitch @doronwolf @avivu 
Maybe we should 'hide' this experiment for a while? Instead of removing it. In Pro we still use the `function is_feature_active()`, which will fail if the experiment is totally removed. Then after a few versions we can delete this.

Reply:
Hi @louiswol94 I understand the reason but wonder, can't the fact there are 2 places that affect the Swiper library version cause other issues/conflicts?

Reply:
@doronwolf The experiment will not be visible to regular users. so regular users will see it only in the settings.